modules/ROOT/pages/index.adoc

```diff
@@ -34,8 +34,8 @@ Mule applications, and adapt APIs to your business needs.
 
 Mule includes an embedded API Gateway, which enables you to apply
 security policies to an API, enrich incoming or outgoing messages, and add
-capabilities to an API without having to write any code. See
-xref:api-manager::api-gateway-capabilities-mule4.adoc[API Gateway Capabilities]
+capabilities to an API without having to write any code. +
+See xref:api-manager::api-gateway-capabilities-mule4.adoc[API Gateway Capabilities]
```

Comment:
"See xref:api-manager::api-gateway-capabilities-mule4.adoc[API Gateway Capabilities] for information about the Mule runtime engine in the API Manager documentation."
My goal as a user is not to understand how the API Manager documentation describes the Mule runtime engine, but to understand something about the engine. The description is not telling me that I'll learn what I want to know or that I'll learn something relevant to me. So, I'm unlikely to click the link.